packages/playwright/src/runner/testServer.ts

```diff
@@ -111,11 +119,12 @@ class Dispatcher {
   }
 
   private async _listTests(reporterPath: string, locations: string[] | undefined) {
-    this._config.cliArgs = [...(locations || []), '--reporter=null'];
-    const reporter = new InternalReporter(new Multiplexer(await createReporters(this._config, 'list', [[reporterPath]])));
-    const taskRunner = createTaskRunnerForList(this._config, reporter, 'out-of-process', { failOnLoadErrors: true });
-    const testRun = new TestRun(this._config, reporter);
-    reporter.onConfigure(this._config.config);
+    const config = await this._loadConfig({});
+    config.cliArgs = [...(locations || []), '--reporter=null'];
+    const reporter = new InternalReporter(new Multiplexer(await createReporters(config, 'list', [[reporterPath]])));
```

Comment:
Why not use `additionalReporters` in the same way here?

Reply:
because they are additional and not overriding.